apps/condo/domains/organization/schema/OrganizationLink.js

```diff
@@ -50,6 +51,16 @@ const OrganizationLink = new GQLListSchema('OrganizationLink', {
         delete: false,
         auth: true,
     },
+    hooks: {
+        validateInput: async ({ resolvedData, addValidationError }) => {
+            if (resolvedData['from']) {
+                const org = await getById('Organization', resolvedData['from'])
+                if (!org || org.type !== HOLDING_TYPE) {
+                    addValidationError(`Parent organization ("from") must have "${HOLDING_TYPE}" type`)
```

Comment:
you should use `GQLError`

Reply:
It's better to use our own GQL messages to help others developers.

Reply:
I hate them, but for your sake I will add one...